Sources/LanguageServerProtocol/FoldingRange.swift

```diff
@@ -21,73 +21,69 @@
 ///
 /// - Returns: A list of folding ranges for the given document.
 public struct FoldingRangeRequest: TextDocumentRequest, Hashable {
-  public static let method: String = "textDocument/foldingRange"
-  public typealias Response = [FoldingRange]?
+      public static let method: String = "textDocument/foldingRange"
+      public typealias Response = [FoldingRange]?
 
-  public var textDocument: TextDocumentIdentifier
+      public var textDocument: TextDocumentIdentifier
 
-  public init(textDocument: TextDocumentIdentifier) {
-    self.textDocument = textDocument
-  }
+      public init(textDocument: TextDocumentIdentifier) {
+            self.textDocument = textDocument
+      }
 }
 
 /// A single folding range result.
 public struct FoldingRange: ResponseType, Hashable {
 
-  /// The zero-based line number from where the folded range starts.
-  public var startLine: Int
+      /// The zero-based line number from where the folded range starts.
+      public var startLine: Int
 
-  /// The zero-based character offset from where the folded range starts.
-  /// If not defined, defaults to the length of the start line.
-  public var startUTF16Index: Int?
+      /// The zero-based character offset from where the folded range starts.
+      /// If not defined, defaults to the length of the start line.
+      public var startUTF16Index: Int?
 
-  /// The zero-based line number where the folded range ends.
-  public var endLine: Int
+      /// The zero-based line number where the folded range ends.
+      public var endLine: Int
 
-  /// The zero-based character offset before the folded range ends.
-  /// If not defined, defaults to the length of the end line.
-  public var endUTF16Index: Int?
+      /// The zero-based character offset before the folded range ends.
+      /// If not defined, defaults to the length of the end line.
+      public var endUTF16Index: Int?
 
-  /// Describes the kind of the folding range such as `comment' or 'region'. The kind
-  /// is used to categorize folding ranges and used by commands like 'Fold all comments'.
-  public var kind: FoldingRangeKind?
+      /// Describes the kind of the folding range such as `comment' or 'region'. The kind
+      /// is used to categorize folding ranges and used by commands like 'Fold all comments'.
+      public var kind: FoldingRangeKind?
 
-  public init(
-    startLine: Int,
-    startUTF16Index: Int? = nil,
-    endLine: Int,
-    endUTF16Index: Int? = nil,
-    kind: FoldingRangeKind? = nil)
-  {
-    self.startLine = startLine
-    self.startUTF16Index = startUTF16Index
-    self.endLine = endLine
-    self.endUTF16Index = endUTF16Index
-    self.kind = kind
-  }
+      public init(
+            startLine: Int, startUTF16Index: Int? = nil, endLine: Int,
+            endUTF16Index: Int? = nil, kind: FoldingRangeKind? = nil
+      ) {
```

Comment:
`) {` on its own line makes me sad 😢

Reply:
That is a standard idiom in the standard library style that this formatter was designed to support.  I'm sorry you don't like it, but it's not just a random side-effect of other rules; it's an intended outcome.

When discretionary breaks are re-enabled, this would also be an allowed result as long as the closing paren fit at the end of the previous line:

```swift
      public init(
            startLine: Int, startUTF16Index: Int? = nil, endLine: Int,
            endUTF16Index: Int? = nil, kind: FoldingRangeKind? = nil)
{
```